package.json

```diff
@@ -7,7 +7,7 @@
     "pythia": "./bin/pythia"
   },
   "scripts": {
-    "test": "ava *.test.js modules/*.test.js",
+    "test": "ava *.test.js modules/*.test.js --serial",
```

Comment:
would prefer not to have to run the tests serially to keep run time as low as possible?

Reply:
Unfortunately the test suite will only run like this because of the use of sinon, see: https://github.com/avajs/ava/issues/1066